app/javascript/App/bundles/ExchangeListings/components/ListingItem.jsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react'
+
+const ListingItem = (props) => {
+  const { listing } = props
```

Comment:
FYI if you want to destructure you can do this in args:

```js
const ListingItem = ({ listing }) => (
  <div />
);
```

Tho I prefer to just use `props` and `props.listing` all the way thus you don't have to worry about anything when prop gets added/removed. But it's really just personal preference.